openapi/saps4hana.itcm.currency/Package.md

```
@@ -1,13 +1,13 @@
-Connects to [SAPS4HANA Intelligent Trade Claims Management(ITCM)](https://help.sap.com/viewer/902b9d277dfe48fea582d28849d54935/CURRENT/en-US) from Ballerina
+Connects to [SAP S/4HANA Intelligent Trade Claims Management(ITCM)](https://help.sap.com/viewer/902b9d277dfe48fea582d28849d54935/CURRENT/en-US) from Ballerina
 
 ## Package overview
-The `ballerinax/saps4hana.itcm.currency` is a [Ballerina](https://ballerina.io/) connector for Currency in SAPS4HANA Intelligent Trade Claims Management.
+The `ballerinax/saps4hana.itcm.currency` is a [Ballerina](https://ballerina.io/) connector for Currency in SAP S/4HANA Intelligent Trade Claims Management.
 
 ### Compatibility
 |                    | Version                   |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)